### PR TITLE
Alex poly

### PR DESCRIPTION
**Pull request by pirolster**
Hey gang, this is my first version of Poly so you don't have to actually merge it into 'develop'. I just created a pull request so that I could assign people to review it. Here are improvements I'm aware of:

1. unit tests
2. auto-doc compatible comments
3. conforming code to our API <-- could use some help with this
4. currently, I get an error that says "no file or directory 'Malli_80s.allASC.ASC'" if I try to use that string as input

Beyond this stuff, any other feedback you have would be super helpful! @JordanHamel, does this seem like the right info to be outputting for the database look-up side of things?

### CODE REVIEW

File: Code/Poly.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# imports
+import matplotlib.pyplot as plt
+import numpy as np
+from numpy.polynomial import Polynomial
+import scipy.interpolate as interpolate
+
+# this function accepts the string name of the XRD data file as an input argument
+# and uses spline-fitting to output the peak locations and heights as an 
+# (Nx2) matrix where N is the number of peaks
+def get_peaks(raw_data):
```
Comment (JordanHamel) [inline Code/Poly.py:10]: Can you write tests for this?

File: Code/Poly.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    deriv = spline.derivative()
+
+    # Find the zeros of the derivatives and the intensity values at these locations
+    roots = deriv.roots()
+    peaks = np.transpose(np.array([roots, spline(roots)]))
+    
+    return peaks
+
+
+    # Plot for sanity check
```
Comment (JordanHamel) [inline Code/Poly.py:47]: I'd prefer if we didn't have commented out code here. If it's not being used, we should just remove it

Comment (pirolster) [reply Code/Poly.py:47]: Agree, I'll take it out. This is just a rough version I wanted to push to Git so ppl could see

**Comment by pirolster** [comment]
Closing and re-creating pull request

**Pull request closed without merging**